philistine/mne/io.py

```diff
@@ -95,28 +97,24 @@ def _write_vmrk_file(vmrk_fname, eeg_fname, events):
         print(r'; Commas in type or description text are coded as "\1".', file=fout)
         print(r'Mk1=New Segment,,1,1,0,0', file=fout)
 
-        twidth = int(np.ceil(np.log10(np.max(events[:,2]))))
+        twidth = int(np.ceil(np.log10(np.max(events[:, 2]))))
         tformat = 'S{:>' + str(twidth) + '}'
 
         for i, r in enumerate(range(events.shape[0]), start=2):
-            print(r'Mk{}=Stimulus,{},{},1,0'.format(i,
-                                                   tformat.format(events[r,2]),
-                                                   events[r, 0]), file=fout)
+            print(r'Mk{}=Stimulus,{},{},1,0'.format(i, tformat.format(events[r, 2]), events[r, 0]), file=fout)
+
 
 def _write_vhdr_file(vhdr_fname, vmrk_fname, eeg_fname, raw,
                      orientation='multiplexed', format='binary_float32'):
-    """ Write BrainvVision marker file)
-    """
+    """ Write Brainvision header file."""
 
     if orientation.lower() not in supported_orients:
         raise ValueError('Orientation {} not supported.'.format(orientation) +
-                         'Currently only these orientations are supported: {}'.format(
-                ', '.join(supported_formats)))
+                         'Currently only these orientations are supported: {}'.format(', '.join(supported_formats)))
 
     if format.lower() not in supported_formats:
         raise ValueError('Data format {} not supported.'.format(format) +
-            'Currently only these formats are supported: {}'.format(
-                ', '.join(supported_formats)))
+                         'Currently only these formats are supported: {}'.format(', '.join(supported_formats)))
```

Comment:
Check out out the way I did this in the new code. In addition to constructing the error message before the `ValueError` constructor, you can also get rid of the second `str.format()` call and just use the concatenation operator `+` since this is the last thing in the string. This also reducesthe number of closing parentheses in a row.